src/main/java/com/synopsys/integration/alert/provider/blackduck/issues/BlackDuckProviderIssueHandler.java

```diff
@@ -59,6 +59,7 @@ public void createOrUpdateBlackDuckIssue(String bomComponentVersionIssuesUrl, Bl
             issueRequestModel.setIssueDescription(existingIssue.getIssueDescription());
             issueRequestModel.setIssueCreatedAt(existingIssue.getIssueCreatedAt());
             issueRequestModel.setIssueUpdatedAt(currentDate);
+            bomComponentVersionIssuesUrl = existingIssue.getMeta().getHref();
```

Comment:
If bomComponentVersionIssuesUrl is passed in, why are we overriding it here? Can you add a comment to explain? Or create a new variable to pass into performRequest(...) that would reveal the intent?

Reply:
A PUT requires the id be appended to the request, but the only way the API reveals that is through the href. I'll update the variable names to make it more clear.